src/EnergyPlus/MixedAir.cc

```diff
@@ -2619,6 +2642,8 @@ namespace MixedAir {
         //		if ( BeginDayFlag ) {
         //		}
 
+        if (OutsideAirSys(OASysNum).AirLoopDOASNum > -1) return;
```

Comment:
There's a lot of these loop number flags floating around...I'm not able to track them fully at this point.

Reply:
@Myoldmopar I know. The main purpose is for by-pass, when AirLoopDOAS is used. In order to reuse existing code, I need to use by-passes to meet AirLoopDOAS requirements.